global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -36,58 +36,42 @@ data:
       - receiver: dev-null
         continue: false
         match_re:
-          region: staging|k-staging|s-staging|lab
-      # routing per tier to <tier>-{info, warning, critical}
-      {{- range $tier := .Values.tiers }}
-      - receiver: {{ $tier }}-info
-        continue: true
+          region: .*staging|lab
+
+      - receiver: slack_by_tier_and_severity
         match_re:
-          tier: {{ $tier }}
-        routes:
-        - receiver: '{{ $tier }}-warning'
-          continue: true
-          match:
-            severity: warning
-        - receiver: '{{ $tier }}-critical'
-          continue: true
-          match:
-            severity: critical
-      {{- end }}
-      # routing per service to openstack-<service>
-      {{- range $service := .Values.openstack_services }}
-      - receiver: openstack-{{ $service }}
-        continue: true
+          tier: openstack|kubernetes|kubernikus|baremetal
+          severity: info|warning|critical
+
+      - receiver: slack_by_tier_and_service
         match_re:
-          service: {{ $service }}
           tier: openstack
-      {{- end }}
-      # Don’t page for qa-de-1
-      - receiver: dev-null
-        continue: false
+          severity: info|warning|critical
+          service: arc|backup|barbican|cinder|designate|elektra|elk|ironic|keystone|limes|lyra|maia|manila|neutron|nova|swift
+
+      - receiver: pagerduty_baremetal
         match_re:
-          region: qa-de-1|k-qa-de-1|s-qa-de-1|lab
+          severity: critical
+          region: [^.*qa-de-1]
+          tier: baremetal
 
-      # block kubernikus and interconnect alerts away from pagerduty
-      - receiver: dev-null
-        continue: false
-        match:
-          tier: kubernikus
-      - receiver: dev-null
-        continue: false
-        match:
-          service: interconnect
+      - receiver: pagerduty_hypervisor
+        match_re:
+          severity: critical
+          region: [^.*qa-de-1]
+          service: vcenter
 
-      {{- range $pd := .Values.pagerduty_services }}
-      - receiver: pagerduty-{{ $pd.name }}
-        continue: false
-        match:
+      - receiver: pagerduty
+        match_re:
           severity: critical
-          {{- if $pd.match }}
-          {{- range $key, $value := $pd.match }}
-          {{ $key }}: {{ $value }}
-          {{- end }}
-          {{- end }}
-      {{- end }}
+          region: [^.*qa-de-1]
```

Comment:
```suggestion
          region: [^.*qa-de-1|lab|.*staging|s-.*|k-.*]
```

Reply:
Maybe it's better to not negate it? To make it very clear what is being forwarded:

```
eu-de-1|eu-de-2|eu-nl-1|...
```